manifests/agent.pp

```diff
@@ -308,6 +323,7 @@
   String $additional_service_params               = $zabbix::params::additional_service_params,
   String $service_type                            = $zabbix::params::service_type,
   Boolean $manage_startup_script                  = $zabbix::params::manage_startup_script,
+  Optional[Boolean] $purge_templates              = $zabbix::params::agent_purge_templates,
```

Comment:
Optional around a boolean is an anti pattern that we try to avoid. Can't we just default to true or false?

Reply:
I think this parameter is also missing in the documentation?